providers/ldap/ldap_client.go

```diff
@@ -165,6 +166,23 @@ func (l *LClient) GenerateToken(jsonInput map[string]string) (model.Token, int,
 			query = groupQuery
 		}
 
+		search := ldap.NewSearchRequest(l.Config.Domain,
+			ldap.ScopeWholeSubtree, ldap.NeverDerefAliases, 0, 0, false,
+			query,
+			searchConfig.UserSearchAttributes, nil)
+		result, err := lConn.Search(search)
+		if err != nil {
+			return nilToken, status, err
+		}
+
+		if len(result.Entries) < 1 {
+			log.Errorf("Cannot locate user information for %s", search.Filter)
+			return nilToken, 403, errors.New("Account does not have access")
+		} else if len(result.Entries) > 1 {
+			log.Error("More than one result")
```

Comment:
If i was a sysadmin and was this error message in the logs, I'd be confused because it doesnt have enough context.

Also, an Error level log message tells sysadmins that something is wrong with the system. They often graph errors to monitor system health and even alert on them to have someone review what's wrong. In that context, do you still think these should be logged as errors?